Assignment/solution_student/css/style.css

```diff
@@ -27,15 +27,179 @@ html {
 .clearfix:after {
   clear: both;
 }
-/******************************************
-/* BASE STYLES
-/*******************************************/
+
 body {
-  color: black;
-  font-size: 16px;
+  font-size: 13px;
+  font-weight: lighter;
   background: white;
+  padding: 50px;
+  color: #05080C;
+  line-height: 1.5;
+  max-width: 900px;
+  margin: 70px auto;
+  background: #F6F6F6;
+  border-top: 5px solid #222222;
+  font-family: 'open sans',sans-serif;
+  border-bottom: 5px solid #222222;
+  padding-top: none;
```

Comment:
padding only takes number values, so what you're looking for isn't `none`, but `0`.